bogo/config.json

```diff
@@ -33,6 +33,8 @@
     "DuplicateCertCompressionExt*": "",
     "TLS-ECH-*": "",
     "ALPS-*": "",
+    "Kyber*": "",
+    "*Kyber*": "",
```

Comment:
Do we need both of these? Wouldn't `*Kyber*` match the same things as `Kyber*` or am I missing a detail?

Reply:
You're completely right -- will clean this up.